src/birthday/birthday.go.tmpl

```diff
@@ -11,7 +11,7 @@
 {{/* DONT TOUCH */}}
 {{/* Vars */}}
 {{$isMod := false}}{{$map := ""}}{{$error := ""}}{{$day := 0}}{{$month := 0}}{{$year := 0}}{{$isUnderAge := false}}{{$isValidDate := false}}{{$user := .User}}{{$checkDate := ""}}{{$insideMap := sdict}}{{$list := cslice}}{{$out := ""}}{{$send := false}}{{$userMonth := ""}}{{$today := sdict}}{{$delay := 86400}}{{$return := sdict}}
-{{$prefix := index (split .Cmd "") 0}}
+{{$prefix := index (split (print .Cmd) "") 0}}
```

Comment:
is this necessary?

Reply:
Yes, it's been causing error for a lot users.

Reply:
Really weird; `.Cmd` should always be a string. Gonna merge this for now but this warrants some further investigation since something is definitely off here and this is just a hotfix.